pkg/scheduler/cache/status_updater/concurrency_test.go

```diff
@@ -252,4 +305,11 @@ var _ = Describe("Status Updater Concurrency", func() {
 			wg.Wait()
 		})
 	})
+
+	Context("Payload processing", func() {
+		It("pod group payload should be applied only once", func() {
+
+		})
+
```

Comment:
Can you populate this test case please?

Reply:
I left this by mistake. This is covered by "Podgroup updates are not moved to applied before the payload processing is done".
Removed stub